packages/graphqlsp/src/diagnostics.ts

```diff
@@ -456,7 +456,7 @@ const runDiagnostics = (
           ) as OperationDefinitionNode;
           if (!op.name) {
             graphQLDiagnostics.push({
```

Comment:
Should this be off for `gql.tada`?

Reply:
No, I honestly still think that providing no operation-name is a bad practice